.github/workflows/phpunit.yml

```diff
@@ -82,7 +73,7 @@ jobs:
           ./psh.phar init-test-databases
           ./psh.phar cache
 
-      # TODO: Re-enable the deprecation helper in SW-497.
+      # Deprecation Helper disabled until https://github.com/shopware/platform/pull/1759 is merged.
```

Comment:
Isn't it merged now? 👍

Reply:
Yeah well, they merged it into `trunk`, which means that the comment should actually reflect `... when 6.4 is available`.